luaa.c

```diff
@@ -67,51 +67,82 @@ extern const struct luaL_Reg awesome_root_lib[];
 extern const struct luaL_Reg awesome_mouse_methods[];
 extern const struct luaL_Reg awesome_mouse_meta[];
 
-/** A call into the lua code aborted with an error
+/** A call into the lua code aborted with an error.
```

Comment:
If you look at old comments, e.g. for luaA_sync / @function sync (or in kill), you'll see that is doesn't seem to be done generally. I usualy look into the style of the source code file I modify and try to adapt to the style used therein, and so I also didn't add empty lines.

Still, I'll add the empty line.